collectors/mimecast/utils.js

```diff
@@ -40,11 +40,14 @@ function getAPILogs(authDetails, state, accumulator, maxPagesPerInvocation) {
                     pageCount++;
                     switch (state.stream) {
                         case Siem_Logs:
-                            if (body.data) {
+                            if (body.data && body.data[0]) {
```

Comment:
is it possible to have `body.data` as an empty array? If yes then we'll have an error addressing the first element.

Reply:
if isLastToken true then it's possible, And I alreday added testcases for same and it's working

Reply:
Done